src/utils/flags/useUDNEnabledFlag.ts

```diff
@@ -0,0 +1,12 @@
+import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { SetFeatureFlag, useK8sModel } from '@openshift-console/dynamic-plugin-sdk';
+import { UserDefinedNetworkModel } from '@utils/models';
+import { isEmpty } from '@utils/utils';
+
+import { FLAG_UDN_ENABLED } from './consts';
+
+export const useUDNEnabledFlag = (setFeatureFlag: SetFeatureFlag) => {
+  const [udnModel] = useK8sModel(modelToGroupVersionKind(UserDefinedNetworkModel));
```

Comment:
Any reason you don't use the UserDefinedNetworkModelGroupVersionKind const you created?